package.json

```diff
@@ -15,7 +15,8 @@
     "clean_modules": "rm -rf node_modules && find ./packages -type d -maxdepth 2 -name \"node_modules\" -exec rm -r {} +",
     "test": "wsrun -p $(yarn --silent run pkgparse) --fast-exit --parallel --no-prefix --exclude-missing --timeout 20000 test",
     "build": "wsrun -p $(yarn --silent run pkgparse) -r --fast-exit --stages --exclude-missing build",
-    "all": "yarn clean && yarn build && yarn test && yarn fix && yarn lint"
+    "all": "yarn clean && yarn build && yarn test && yarn fix && yarn lint",
+    "test:docker": "docker-compose -f docker-compose.local.yml up --abort-on-container-exit --exit-code-from integration_tests"
```

Comment:
This accidentally got in from my attempts at running this locally. I wrote a bash script to build the containers and run them, but I think i had something misconfigured because the postgres image would crash immediately. Went with pushing to github to have the ci run in the cloud instead.

Reply:
Got it. We should definitely figure out how to easily run these integration tests locally / put it in the readme